src/patternfly/components/Switch/switch-input.hbs

```diff
@@ -1,13 +1,19 @@
 <input class="pf-c-switch__input{{#if switch-input--modifier}} {{switch-input--modifier}}{{/if}}"
   type="checkbox"
   id="{{id}}"
-  aria-labelledby="switch-is-{{id}}"
+  aria-label="{{id}}"
   {{#if switch-input--attribute}}
     {{{switch-input--attribute}}}
   {{/if}}
-  {{#if checked}}
-    {{{switch-label-on}}}
+  {{#if aria-labelledby}}
   {{else}}
-    {{{switch-label-off}}}
+    {{#if checked}}
+      aria-label="switch is on"
+    {{else}}
+      aria-label="switch is off"
+    {{/if}}
+  {{/if}}
+  {{#if switch-input--attribute}}
+    {{{switch-input--attribute}}}
```

Comment:
It looks like you still need to rebase against the latest changes made to the switch.